package.json

```diff
@@ -50,8 +50,7 @@
     "freedom": "^0.6.11"
   },
   "scripts": {
-    "test": "grunt test",
-    "prepublish": "grunt test",
+    "test": "grunt build jasmine",
```

Comment:
OK, that makes sense. How about we make a `unit-tests` target and an `integration-tests` target, and each of those runs the relevant tests, and then it'll be super clear what's happening. :)